src/awx_plugins/credentials/azure_kv.py

```diff
@@ -1,11 +1,28 @@
-# FIXME: the following violations must be addressed gradually and unignored
-# mypy: disable-error-code="import-untyped, no-untyped-def"
+"""Microsoft Azure Key Vault Lookup Plugin.
+
+This module defines a credential lookup plugin to authenticate and retrieve
+secrets from an Azure Key Vault. If the Client ID, Tenant ID, and Client
+Secret are provided it will create a credential with those. If one is missing,
+ it will attemmpt to use the Managed Identity of an Azure VM to create a
```

Comment:
I think RST might be unhappy about this indent/leading whitespace, as it starts the block quote.
```suggestion
it will attemmpt to use the Managed Identity of an Azure VM to create a
```